src/elexclarity/convert.py

```diff
@@ -22,7 +23,18 @@ def convert(data, statepostal=None, level=None, outputType="results", style="def
         data = [xmltodict.parse(data, attr_prefix="")["ElectionResult"]]
 
     if level == "precinct" or level == "county":
-        county_fips_lookup = {v["name"]: k for k, v in STATE_COUNTIES[statepostal].items()}
+        # If specific county mapping is provided
+        if countymapping:
+            fips_mapping = countymapping[statepostal]
```

Comment:
If a mapping is getting passed in, I think we should assume it's already formatted like `{ <Clarity county name> : <county identifier> }` (the reversal that's happening here is kind of specific to the elex-static-data format)